models/testdevice-1.0.x/go.mod

```diff
@@ -12,7 +11,5 @@ require (
 	github.com/openconfig/gnmi v0.0.0-20210914185457-51254b657b7d
 	github.com/openconfig/goyang v0.4.0
 	github.com/openconfig/ygot v0.14.0
-	github.com/stretchr/testify v1.7.0
```

Comment:
i think these fall out if you only run `make models` and they appear back again if you run `make models-openapi` or `make test` - maybe we should add them as `_` deps in the `main.go.tmpl` template